### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by yeaungpyay** [community/community#70524] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how do i unsubsciribe

**Comment by cmdhell** [comment]
You need to go on the channel which you want to unsubsciribe and click on it and then You will unsubsciribe it.

**Reply by yeaungpyay** [reply]
i don't think i have any channels
is there another way to unsubscribe.

**Comment by yeaungpyay** [comment]
Thanks,
i think i've done it.
regards

**Comment by ghostinhershell** [comment]
Hi @yeaungpyay, 

It looks like you've already found the answer to your question. Regardless, I'd like to share this support article so that anyone who may also have this question will be able to reference this as well: [Managing your subscriptions](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions).

Let us know if you have any additional questions. Cheers!

**Comment by dubravko-luka** [comment]
Reponse #10087
